tests/robustness/validate/validate.go

```diff
@@ -58,6 +57,12 @@ type Config struct {
 	ExpectRevisionUnique bool
 }
 
+func prepareAndCategorizeOperations(reports []report.ClientReport, persistedRequests []model.EtcdRequest) ([]report.ClientReport, []porcupine.Operation, []porcupine.Operation) {
+	patchedReport := patchFailedRequestWithInfiniteReturnTime(reports)
+	return patchedReport, patchLinearizableOperations(patchedReport, persistedRequests), filterSerializableOperations(patchedReport)
```

Comment:
This should not be `patchLinearizableOperations` but `relevantOperations`

Reply:
Refactored as requested!